Anders

xRipzch opened the pull request:
Minor bug fixed.
Spinner addded to avoid blank pages when quest & questDetails is loading.

sgtasswipe committed:
added github action for deployment

xRipzch committed:
Merge remote-tracking branch 'origin/dev' into Anders

xRipzch committed:
Merge remote-tracking branch 'origin/main' into Anders

xRipzch committed:
Improve styles for newQuest and questDetails pages; update login.html

xRipzch committed:
Improve quest details styling and functionality

- Updated styles for quest pages
- Enhanced quest details layout and scripts

xRipzch committed:
Update styles and HTML for index and signup pages, and improve login functionality

- Modified common.css and index.css for better styling
- Updated index.html and signup.html for improved layout
- Enhanced login.js for better user authentication

heizetagram reviewed (approved):
LGBT 😘 elsker loading spinners

xRipzch committed:
Add navigation and shared quest features

- Updated package.json
- Improved quest details styling in questDetails.css
- Updated index.html, login.html, newQuest.html, and questDetails.html
- Enhanced main.js functionality
- Added navigation.js for navigation features
- Updated questDetails.js with new functionalities
- Added sharedQuest.js for shared quest features

xRipzch committed:
Merge remote-tracking branch 'origin/main' into Anders
